cluster-autoscaler/cloudprovider/aws/README.md

```diff
@@ -1,14 +1,22 @@
 # Cluster Autoscaler on AWS
-The cluster autoscaler on AWS scales worker nodes within any specified autoscaling group. It will run as a `Deployment` in your cluster. This README covers the steps required to configure and run the cluster autoscaler.
+On AWS, Cluster Autoscaler utilizes Amazon EC2 Auto Scaling Groups to manage node
+groups. Cluster Autoscaler runs as a `Deployment` in your cluster.
```

Comment:
Theoretically, couldn't it be deployed elsewhere? I might soften this language to "Cluster Autoscaler can be easily run"

Reply:
How about "typically runs"? I haven't heard of any situations in which a customer might run it elsewhere; have you?

Reply:
You could imagine running it on the kubernetes master or in a cloud provider's control plane :)